python/scripts/meta.json

```diff
@@ -2084,5 +2084,54 @@
             }
         ]
 
+    },
+
+    "reac_scores": {
+        "cleaner": "reac_score_cleaner",
+        "replace_table": true,
+        "fields": [
+            {
+                "display_name": "nlihc_id",
```

Comment:
You can look up info from HUD on how they describe reac scores to give a useful summary - should give users enough context to at least know what to google to find more if they need to.